src/Tribe/Asset/Calendar_Script.php

```diff
@@ -15,6 +15,18 @@ public function handle() {
 			'events_base'        => tribe_get_events_link(),
 		);
 
+		/**
+		 * Allows for easier filtering of the "Export Events" iCal link URL.
+		 *
+		 * @since TBD
+		 *
+		 * @see tribe_get_ical_link
+		 * @param boolean $force Defaults to false; when true, the dynamic JS generation of the "Export Events" URL is disabled.
+		 */
+		if ( apply_filters( 'tribe_force_filtered_ical_link', false ) ) {
```

Comment:
* Hooks inside The Events Calendar should use the `tribe_events_*` prefix [(see here)](http://moderntribe.github.io/products-engineering/guidelines/plugins-namespacing/)
* I love when we add new hooks, but I wonder if it would make more sense to widen the scope and make `$js_config_array` filterable (vs just one key within that array)?

Reply:
> Hooks inside The Events Calendar should use the tribe_events_* prefix (see here)

My bad! Will fix.

> I love when we add new hooks, but I wonder if it would make more sense to widen the scope and make $js_config_array filterable (vs just one key within that array)?

I tried that, but failed to have it reliably work. Most of my clocked time on the ticket actually went to trying to make that work.

While I think making those vars filterable generally is maybe cool, it didn't really work for this specific issue; would be happy to demonstrate on a hangout or something (just because explaining over text would take a while).

Although if you want me to type out why it didn't quite work, I can do that too 😎